src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Aerospike/AerospikeCommon.cs

```diff
@@ -0,0 +1,86 @@
+// <copyright file="AerospikeCommon.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.DuckTyping;
+using Datadog.Trace.Tagging;
+using Datadog.Trace.Vendors.Serilog;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Aerospike
+{
+    internal class AerospikeCommon
+    {
+        private const string ServiceName = "aerospike";
+        private const string OperationName = "aerospike.command";
+        public const string IntegrationName = nameof(IntegrationIds.Aerospike);
+        internal static readonly IntegrationInfo IntegrationId = IntegrationRegistry.GetIntegrationInfo(IntegrationName);
+
+        public static Scope CreateScope<TTarget>(Tracer tracer, out AerospikeTags tags, TTarget target)
```

Comment:
AFAICT, the `out tags` are never used - does it need to be an `out` parameter?

Reply:
- [ ] You're right. It bugged me during the night, then I forgot again in the morning 😄